### DISCUSSION TITLE
Disabling/lowering caching on RSS/Atom feeds

### DISCUSSION THREAD

**Discussion by ianpmurphy** [grafana/grafana#64500] [Dashboards, Panels & Visualizations]
Is there any way to either disable caching completly or lowering the cache time for data pulled in via the RSS/Atom panel? 
I have a sinage system set up to display various graphs and data and they all work well. However we have several panels configured to display 'news' which take ages to update. By ages I mean it can take more than a day for an update to appear.

I really don't care if the panel downloads the rss feed every single time. I'm running the sinage on a dedicated Raspberry PI and it downloads 'news' from our own server on the local lan and its just a tiny text file.

Any suggestions on how to fix this? In the past I have modified Grafana modules and rebuilt them, but only tiny changes. In this case I can't find where this is being cached or where the problem is.
Any pointers/help gratefully received.

**Comment by gelicia** [comment]
I'm not entirely sure, but I believe you may want to ask this question here https://github.com/VolkovLabs/volkovlabs-rss-datasource
